debian/rules

```diff
@@ -18,7 +18,8 @@ else
 endif
 
 extraopts += -DWITH_OCF=ON -DWITH_LTTNG=ON
-extraopts += -DWITH_PYTHON3=ON -DWITH_MGR_DASHBOARD_FRONTEND=OFF
+extraopts += -DWITH_MGR_DASHBOARD_FRONTEND=OFF
+extraopts += -DWITH_PYTHON3=ON -DWITH_PYTHON2=OFF -DMGR_PYTHON_VERSION=3 -DWITH_MGR_PYTHON3=ON
```

Comment:
```suggestion
extraopts += -DWITH_PYTHON3=3 -DWITH_PYTHON2=OFF -DMGR_PYTHON_VERSION=3
```

Reply:
Alright, I think for real this time I got it right